app/models/user.rb

```diff
@@ -12,4 +12,8 @@ class User
   field :organisation_content_id, type: String
   field :disabled, type: Boolean, default: false
 
+  def is_gds_editor?
```

Comment:
Perhaps `gds_editor?` :blush: Since predicates answer true/false then the `is_` part is superfluous.